src/libraries/System.Numerics.Vectors/tests/Generation/Test/Vector3Tests.cs

```diff
@@ -0,0 +1,1178 @@
+/*********************************************************************************
```

Comment:
License header needs to be first, auto-generated comment likely needs to go away now